src/Components/CourseSelection.js

```diff
@@ -1,9 +1,43 @@
 import '../css/course-selection.css';
 import React from 'react';
-import { Tab, Nav} from 'react-bootstrap';
+import { Tab, Nav, Col, Row} from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMinus } from '@fortawesome/free-solid-svg-icons';
+import { faAngleDown } from '@fortawesome/free-solid-svg-icons';

```

Comment:
Looks pretty good! We could also consider an (info) symbol so it's even clearer that clicking is for info.